cmd/genji/shell/command.go

```diff
@@ -136,3 +140,132 @@ func displaySuggestions(in string) error {
 	fmt.Println()
 	return nil
 }
+
+// dumpTable display the content of the given table as SQL statements.
+func dumpTable(db *genji.DB, t *database.Table) error {
+	// Get table info.
+	ti, err := t.Info()
+	if err != nil {
+		return err
+	}
+
+	fields := ti.FieldConstraints
+	if len(fields) == 0 {
+		fmt.Printf("CREATE TABLE %s;\n", t.Name())
```

Comment:
We should use a [strings.Builder](https://golang.org/pkg/strings/#Builder) instead of printing chunks. If there is an error in the middle of printing a statement we might end up with weird output.
Or even a better, a [bytes.Buffer](https://golang.org/pkg/bytes/#Buffer) combined with [fmt.Fprintf](https://golang.org/pkg/fmt/#Fprintf)